proposals/aip-1.md

```diff
@@ -0,0 +1,229 @@
+- Roles:
+  - ONLY the [@Agorex-io/board-of-directors] can approve proposals that move beyond a PR and end up merged to the repository. 
+  - [@Agorex-io/proposal-editors] cannot approve, but can facilitate the process and guide champions and authors. They can also edit PRs in progress to help get the preamble and formatting correct according to the workflow. 
+- Assumes [required reviews have been set up **to include repository admins**](https://help.github.com/articles/enabling-required-reviews-for-pull-requests) option is on.
+- Assumes [Dismiss stale pull request approvals when new commits are pushed](https://help.github.com/articles/enabling-required-reviews-for-pull-requests/) option is on.
+- Assumes that **Board of Directors** is how we refer to the governing body for the community.
+  - I'm using "Board of Directors" as we have referred to it this way most often (and in AIP-2 issue discussions).
+- Assumes the target size for the Board of Directors is 9.
+- Note that the EIP process in EIP-1 is written focused on proposals for features and feels retrofitted for process and informational proposals. I am trying to make this more designed around community/structural proposals first, and expect we'll add a "feature proposal" type later. I think this will be better basis for the process in our case (which is decidedly different circumstance than Ethereum which also maintains a protocol with many clients & implementations).
+
+
+# Open Issues / TODO #
+- [x] ~~Near the mention of "We use GitHub's [required reviews for pull requests]" above, but that is for every review/pull of the PR. PRs can be merged in a Draft status without being accepted. More final status (Accepted, Deferred, Rejected, and Replaced, Final) require more careful reviews. That needs clarified. Or we could make a different directory for Draft proposals and Accepted/Finalized/Active/Replaced proposals to keep the review/approval requirement lower for Draft proposals.~~
+    - It can be difficult to easily preserve and view file history across renames, so I think we just require n of M board approvals for ALL pulls and make sure that they are trained to quite liberally merge draft proposals and very judiciously merge proposals with any other "final" status. This also keeps the URL to an AIP intact regardless of status.
+- [ ] Set up the [@Agorex-io/proposal-editors] team.
+- [ ] Set up the [@Agorex-io/board-of-directors] team.
+  - Initially proposal-editors and board-of-directors are likely to be the same members. Long term this may scale better to keep them separate.
+- [ ] README.md
+- [ ] add PULL_REQUEST_TEMPLATE.md
+- [ ] Document and link to "Agorex values" above.
+- [x] Double check by searching for PEP, EIP and Ethereum in doc content.
+- [x] Double check by searching for Meta -> Proposal in doc content.
+- Low priority:
+  - [ ] Setup github pages on the repo. Necessary?
+
+
+# Future Consideration #
+- Use PGP-signed commits?
+
+
+# References #
```

Comment:
This markdown simply sets those links.

You'll also need to list them like so:

```
[the Issues section of this repository]
[the Agorex subreddit]
[the Agorex Discord chat rooms]
[pull request in this repository]
```

Reply:
@JonathonDunford Right, they are all referenced in the doc above. Do you think it will be best to duplicate each reference as an explicit link so that they are visible in the markdown? Or maybe put a comment to explain why they're only visible in the source markdown itself? Or maybe just remove the "References" heading?

Reply:
Maybe just remove the references heading.

Everything should be viewed from Markdown IMO